Added Day 29 – Animated Login Form

shreyas1103 opened the pull request:
This PR adds a new mini project "Animated Login Form" to the repository under public/Day29.

Related Issue
Closes #60 

Changes made:
  Created Day29/index.html, style.css, and script.js.
  Implemented floating labels with smooth animations.
  Added password show/hide toggle functionality.
  Made the layout responsive for mobile and desktop.
  Updated main project listing (index.html) to include Day 29.

Tech stack:
  HTML
  CSS (transitions, pseudo-classes)
  JavaScript (password visibility toggle)

shreyas1103 commented:
Hi @rutikakengal,
This pull request contains the Day 29 project built using HTML, CSS, and JavaScript for issue #60 . Looking forward to your feedback!

rutikakengal commented:
@shreyas1103 , it is too simple , and u are mentioning animated ! Kindly fix this Issue !

rutikakengal commented:
It's not working in deploy preview , plz fix it @shreyas1103

shreyas1103 commented:
This PR adds a new mini project "Animated Login Form" to the repository under public/Day29.

Related Issue
Closes #60 

Changes made:
  Created Day29/index.html, style.css, and script.js.
  Implemented floating labels with smooth animations.
  Added password show/hide toggle functionality.
  Made the layout responsive for mobile and desktop.
  Updated main project listing (index.html) to include Day 29.

Tech stack:
  HTML
  CSS (transitions, pseudo-classes)
  JavaScript (password visibility toggle)

shreyas1103 commented:
@rutikakengal Now its working in deploy preview....Looking forward to your feedback!
